django/core/cache/backends/redis.py

```diff
@@ -69,11 +72,17 @@ def _get_connection_pool_index(self, write):
 
     def _get_connection_pool(self, write):
         index = self._get_connection_pool_index(write)
-        if index not in self._pools:
-            self._pools[index] = self._pool_class.from_url(
-                self._servers[index],
+        if index in self._pools:
+            if self._pools[index].connection_kwargs == {
                 **self._pool_options,
-            )
+                **parse_url(self._servers[index]),
+            }:
+                return self._pools[index]
+            self._pools = {}
```

Comment:
```suggestion
```

Reply:
@sarahboyce Can you please tag some people you know here?

Reply:
I tagged some people on the PR :+1: 
Can you help me understand why we want to reset `_pools` here?

Reply:
This is for handling a case when someone updates the Cache config dynamically from the code.

Reply:
on second thought this might not be required, because the next line creates a new Connection and assign's it to the pool.

Removing this, and updating the PR.